libirods_rule_engine_plugin-indexing.cpp

```diff
@@ -201,6 +201,14 @@ namespace {
                     object_path,
                     _rei->rsComm->clientUser.userName,
                     source_resource);
+
+                const char* metadata_included = getValByKey(&obj_inp->condInput, METADATA_INCLUDED_KW);
+                if (metadata_included)
+                {
+                    idx.schedule_metadata_indexing_event(
+                        object_path,
+                        _rei->rsComm->clientUser.userName, "a", "v", "u"); // the actual AVU values do not matter since commit 277933912f
```

Comment:
Does the commit referenced in the comment point to a commit in 4-2-stable?

Reply:
referring to new issue instead

Reply:
Please mark as resolved.